### ISSUE TITLE
Web UI not showing active downloads

### ISSUE THREAD

**Issue by yobionekanobi** [Secretmapper/combustion#46]
Saw someone else left a comment on your reddit post about combustion web ui not displaying active torrents. I’m having the same issue as well. No way of only displaying or sorting active torrents as that parameter always shows zero value. Other than this discrepancy, this is by far the best web ui I’ve come across!

**Comment by lilws** [comment]
Yes, I have this problems too. Active here is meaning of active downloading or uploading. Please fix this. It is the best UI so far I found. Clean and easy to use on mobile.
